www/app/App.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import BaseApp from 'next/app';
+import Head from 'next/head';
+import keyboardOnlyOutlines from 'keyboard-only-outlines';
+import ReactGA from 'react-ga';
+import getConfig from 'next/config';
+
+import seoImage from '../shared/media/images/seo.png';
+import favicon from '../shared/media/favicons/favicon.ico';
+
+import '../shared/styles/index.css';
+
+const { publicRuntimeConfig } = getConfig();
+
+const SEO_DATA = {
+    url: 'https://{project-domain}',
+    title: '{project-name}',
+    description: '{project-description}',
+    keywords: ['{project-keyword-1}', '{project-keyword-2}'],
+    image: { src: seoImage, width: 1200, height: 630 },
+};
```

Comment:
Check the comment about _data files_. I suggest placing this properties in a _data file_.